scripts/pre_leap.sh

```diff
@@ -19,6 +19,16 @@
 set -e -u
 set -o pipefail
 
+# Choose if preserve Elasticsearch data in the leap
+read -p 'Enter "YES" to make sure you want to preserve Elasticsearch logs in leapfrog or anything else to skip: ' TO_UPGRADE_ELASTICSEARCH
```

Comment:
Because this prompts for user input, this will break gating scripts.  The openstack-ansible-ops has a VALIDATE_UPGRADE_INPUT which lets you bypass the prompts but that's not at this lower level.  I'd rather we just set default vars for those two here, and allow the user to override if they don't want those done.

Reply:
Cool, just export vars to execute es-upgrade, and let user override them if they don't like.